lemonparty2018.py

```diff
@@ -1,11 +1,23 @@
 from flask import Flask
 from flask import render_template
-from localsettings import DEBUG
+from localsettings import DEBUG, VERSIONS
 
 
 app = Flask(__name__)
 
 
+@app.context_processor
+def static_path_processor():
+    if DEBUG:
+        css_path = 'app.css'
+        js_path = 'app.js'
+    else:
+        css_path = 'app.min.css'
+        js_path = 'app.min.js'
+
+    return dict(css_path=css_path, js_path=js_path, VERSIONS=VERSIONS)
```

Comment:
I would use the more standard python dictionary syntax here: 
`{'js_path': 2, 'css_path': 1, 'VERSIONS': 3}`

Reply:
👍